### PR TITLE
Bug 1125878 — Time calls to TokBox.

### PR DESCRIPTION
**Pull request by ametaireau**

### CODE REVIEW

**Comment by ametaireau** [comment]
@n1k0 r?

**Comment by ametaireau** [comment]
actually I wanted to direct this to @Natim. r?

File: test/tokbox_test.js [hunk 1]
```diff
@@ -71,9 +71,12 @@ describe("TokBox", function() {
   });
 
   describe("#getSessionTokens", function() {
-    var tokBox, openTokSpy;
+    var tokBox, openTokSpy, statsdClient, statsdSpy;
 
     beforeEach(function() {
+      statsdClient = { timing: function() {} };
+      statsdSpy = sandbox.spy(statsdClient,"timing");
```
Comment (Natim) [inline test/tokbox_test.js]: space after `,`

**Comment by Natim** [comment]
LGTM

**Pull request merged**